tests/UI/campaigns/functional/BO/00_login/01_passwordReminder.js

```diff
@@ -0,0 +1,273 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+const mailHelper = require('@utils/mailHelper');
+
+// Importing pages
+// BO pages
+const loginPage = require('@pages/BO/login/index');
+const dashboardPage = require('@pages/BO/dashboard');
+const emailPage = require('@pages/BO/advancedParameters/email');
+const employeesPage = require('@pages/BO/advancedParameters/team/index');
+const addEmployeePage = require('@pages/BO/advancedParameters/team/add');
+
+// Import datas
+const {DefaultCustomer} = require('@data/demo/customer');
+const EmployeeFaker = require('@data/faker/employee');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_login_passwordReminder';
+
+let browserContext;
+let page;
+let numberOfEmployees = 0;
+// maildev
+let newMail;
+const {smtpServer, smtpPort} = global.maildevConfig;
+const resetPasswordSuccessText = 'Please, check your mailbox.';
+const testMailSubject = 'Test message -- Prestashop';
+const resetPasswordMailSubject = 'Your new password';
+
+// new employee datas
+const createEmployeeData = new EmployeeFaker({
+  defaultPage: 'Products',
+  language: 'English (English)',
+  permissionProfile: 'Salesman',
+});
+
+// init mailListener
+const mailListener = mailHelper.createMailListener();
+
+describe('BO Password reminder', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+    mailHelper.startListener(mailListener);
+    // Handle every new email
+    mailListener.on('new', (email) => {
+      newMail = email;
+    });
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+    mailHelper.stopListener(mailListener);
```

Comment:
```suggestion
    await helper.closeBrowserContext(browserContext);
    
    // Stop listening to maidev server
    mailHelper.stopListener(mailListener);
```

Just to improve readability